src/common/utxobased/info/litecoin.ts

```diff
@@ -6,10 +6,12 @@ import { maximumFeeRateCalculator } from '../../plugin/util/maximumFeeRateCalcul
 import { memoInfo } from './commonInfo'
 
 export const currencyInfo: EdgeCurrencyInfo = {
-  pluginId: 'litecoin',
-  walletType: 'wallet:litecoin',
+  canReplaceByFee: false,
```

Comment:
I feel like `canReplaceByFee` field is in a different "group" or category of fields for the EdgeCurrencyInfo. Perhaps it would make sense to put a comment above `canReplaceByFee` and potentially other fields like it and separate it from the required fields like `pluginId` and `currencyCode`.

Reply:
Seeing the following commits after this comment made me realize the change request is too much rebasing effort to be worth it. I'll leave the idea here as optional though.